modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -1688,4 +1688,132 @@ definition {
 		}
 	}
 
+	@description = "This test asserts that the User's available views and actions within the page shall reflect the permissions of the Role in the Selected Account."
+	@priority = "5"
+	test ThePageViewsAndActionsAreConsistentWithRolesPermissionsInSelectedAccount {
```

Comment:
Need re-design the test steps, since the test is about Selected Account, seems there is not steps related to it.

Reply:
done